src/lib/plugin_apis/crypto.api

```diff
@@ -1,5 +1,6 @@
 #include <glib.h>
 #include <blockdev/utils.h>
+#include <libcryptsetup.h>
```

Comment:
I wish this was not needed, but I guess it's better than having some magic numbers in the `enum` definition below.